config/prod.exs

```diff
@@ -226,6 +226,12 @@ config :archethic, ArchethicWeb.FaucetController,
 # which you should run after static files are built and
 # before starting your production server.
 config :archethic, ArchethicWeb.Endpoint,
+  explorer_url:
```

Comment:
You should also take care of the testnet possibliity.
For this you can leverage `ARCHETHIC_NETWORK_TYPE` env variable.

So in short:
- if network type is testnet
   - take the `ARCHETHIC_DOMAIN_NAME`
   - if not defined, use `https://testnet.archethic.net`
- else
   - take the `ARCHETHIC_DOMAIN_NAME`
   - if not defined, use `https://mainnet.archethic.net`